spec/requests/api/v1/countries/destroy_spec.rb

```diff
@@ -0,0 +1,31 @@
+describe 'DELETE /api/v1/countries/:id', type: :request do
+  subject { delete api_v1_country_path(country.id), headers: headers, as: :json }
+
+  let(:headers) { auth_headers }
+  let(:user) { create(:user) }
+
+  before { subject }
+
+  context 'when response is successful' do
+    let(:country) { create(:country) }
+
+    it 'returns no content status code' do
+      expect(response).to be_no_content
+    end
+  end
+
+  context 'when response is an error' do
+    context 'when user not logged in' do
+      let(:headers) { nil }
+      let(:country) { create(:country) }
+
+      it_behaves_like 'an unauthorised request'
+    end
+
+    context 'when country does not exist' do
+      let(:country) { build(:country, id: 1) }
```

Comment:
To avoid this `let`, what about this:

```ruby
  subject { delete api_v1_country_path(country_id), headers: headers, as: :json }

  let(:country) { create(:country) }
  let(:country_id) { country.id }

  context 'when country does not exist' do
    let(:country_id) { 'some_random_and_not_real_id' }
  end
```

Reply:
Updated